README.md

```diff
@@ -289,219 +289,24 @@ $ sudo yum install -y gnupg2 pinentry-curses pcsc-lite pcsc-lite-libs gnupg2-smi
 
 ## NixOS
 
-Generate an air-gapped NixOS LiveCD image with the given config:
-
-```nix
-# yubikey-installer.nix
-let
-  configuration = { config, lib, pkgs, ... }:
-    with pkgs;
-    let
-      src = fetchGit "https://github.com/drduh/YubiKey-Guide";
-
-      guide = "${src}/README.md";
-
-      contrib = "${src}/contrib";
```

Comment:
These can just be relative paths to this repo and they'll end up in the nix store. E.g

```nix
  guide = ./README.md;
  contrib = ./contrib;
```